src/core/execute.c

```diff
@@ -2146,6 +2146,7 @@ static int setup_exec_directory(
         return r;
 }
 
+#if ENABLE_SMACK
```

Comment:
We could also use `_unused_` which is a bit more lightweight. But `#if` works too.